apps/extension/src/ui/apps/popup/pages/Portfolio/PortfolioWhatsNew.tsx

```diff
@@ -0,0 +1,111 @@
+import { HandMonoLogo } from "@talisman/theme/logos"
+import { ChevronLeftIcon } from "@talismn/icons"
+import { AnalyticsPage, sendAnalyticsEvent } from "@ui/api/analytics"
+import { useSetting } from "@ui/hooks/useSettings"
+import { useCallback } from "react"
+import { Trans, useTranslation } from "react-i18next"
+import { useNavigate } from "react-router-dom"
+
+import HeroUrl from "./assets/Whats New - Hero.png"
+
+const ANALYTICS_PAGE: AnalyticsPage = {
+  container: "Popup",
+  feature: "Portfolio",
+  featureVersion: 2,
+  page: "What's New",
+}
+
+/**
+ * This number should be incremented when we want to show the `What's New` banner
+ * to users who have dismissed it in the past.
+ *
+ * E.g. when we have new features to tell them about.
+ */
+export const WhatsNewVersion = 1
+
+export const PortfolioWhatsNew = () => {
```

Comment:
It would be great if we could make this component generic, and just have it accept a JSON of values configuring the What's New message according to some standard format. This way anyone (@0byt for example) could submit a PR with a JSON object for a new message.